packages/core/src/controllers/ApiController.ts

```diff
@@ -138,6 +138,16 @@ export const ApiController = {
     }
   },
 
+  getChains() {
+    const { caipNetwork } = NetworkController.state
```

Comment:
`caipNetwork` should only refer to a current connected chain and we should not rely on it for static configuration, we should use `requestedCaipNetworks` instead